src/Cynthia.Card/src/Cynthia.Card.Common/GwentGame/CardId.cs

```diff
@@ -558,5 +558,8 @@ public static class CardId
         public const string CongregationCleric = "70076";
         public const string FireswornZealot = "70077";
         public const string Damnation = "70078";
+        public const string AnCraiteWarlord = "70079";
+        public const string KnutTheCallous = "70080";
+        public const string Drummond Pillager = "70081";
```

Comment:
Drummond Pillager should be together, like  DrummondPillager
```suggestion
        public const string DrummondPillager = "70081";
```